### ISSUE TITLE
[A11y] Improve accessibility for the Carousel block

### ISSUE THREAD

**Issue by jlemay-godaddy** [godaddy-wordpress/coblocks#1961]
**Accessibility problem:**
These are either empty or not accessible

Requested in #1927
